controllers/presentationController.js

```diff
@@ -11,31 +11,69 @@ async function getAllPresentations(req, res) {
     if (!allPresentations) {
       res
         .status(404)
-        .send({ result: "error", error: "Presentations not found" });
+        .json({ message: "No presentations found created by the user" });
```

Comment:
```suggestion
        .json({ result: "error", message: "No presentations found created by the user" });
```
하단 부분에 result 프로퍼티를 함께 전송해주는 코드가 있어서, 모든 응답 전송에 result 프로퍼티가 있도록 통일되게 구현되어야 할 것 같습니다 💯